source/working_with_kayobe.rst

```diff
@@ -51,7 +49,8 @@ See the next section for details.
            --kayobe-repo |kayobe_source_url| \
            --kayobe-branch |kayobe_source_version| \
            --kayobe-config-repo |kayobe_config_source_url| \
-           --kayobe-config-branch |kayobe_config_source_version|
+           --kayobe-config-branch |kayobe_config_source_version| \
+           --vault-password-file |vault_password_file_path|
 
 After making the checkout, source the virtualenv and Kayobe config environment variables:
 
```

Comment:
If we pass in the kayobe environment and vault password to beokay, the rest of this section is redundant. 
Likewise, we can drop the suggestion to update the bash profile and just point to the beokay env-vars.sh script